aspnetcore/mvc/models/file-uploads/samples/3.x/SampleApp/Controllers/StreamingController.cs

```diff
@@ -100,9 +100,12 @@ public async Task<IActionResult> UploadDatabase()
                         trustedFileNameForDisplay = WebUtility.HtmlEncode(
                                 contentDisposition.FileName.Value);
 
-                        streamedFileContent = 
-                            await FileHelpers.ProcessStreamedFile(section, contentDisposition, 
-                                ModelState, _permittedExtensions, _fileSizeLimit);
+                        using (var memoryStream = new MemoryStream())
```

Comment:
This will still blow up over ~2GB.  I only fixed up the file impl, not the EF impl.